testData/solutions/win32.json5

```diff
@@ -6115,6 +6115,70 @@
             }
         ]
     },
+    "com.microsoft.windows.touchPadSettings": {
+        "name": "Windows precision touchpad settings",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.systemSettingsHandler",
+                "capabilities": [],
+                "capabilitiesTransformations": {
+                    "SystemSettings_Input_Touch_SetActivationTimeout": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.touchPadSettings.Sensitivity",
+                            "outputPath": "value"
+                        }
+                    }
+                },
+                "supportedSettings": {
+                    "SystemSettings_Input_Touch_SetActivationTimeout": {
```

Comment:
There's no need to namespace this setting, it is only scoped to this particular settingsHandler's action - SetActivationTimeout is fine

Reply:
I don't really namespace it, it's the actual name of the setting, the systemSettingsHandler expects the actual name of the setting as part of the payload to process.